pkgs/development/tools/continuous-integration/gitlab-runner/default.nix

```diff
@@ -32,6 +32,8 @@ buildGoPackage rec {
     sha256 = "0n8hcj2b1pb95x4bd7fb9ri43vgc4h2dj2v3iiziw2imqjyphfx4";
   };
 
+  patches = [ ./v1-fix-shell-path.patch ];
```

Comment:
My problem with that dirty patch is that it breaks gitlab-runner from nixpkgs for non-NixOS environments. I'm trying to convince some colleagues who are proficient in go to implement shell path lookup through the PATH environment variable. So we hopefully get a proper fix upstream soon. :)